diagnostic_aggregator/test/launch/test_script_add.launch

```diff
@@ -0,0 +1,3 @@
+<launch>
+  <node pkg="diagnostic_aggregator" type="add_analyzers" name="add_analyzers" args="$(find diagnostic_aggregator)/test/multiple_match_analyzers.yaml /test/namespace"/>
```

Comment:
Should result in an easier scrip, and also follows the aggregator style to load the parameters with rosparam rather than using an argument.